app/channel.go

```diff
@@ -767,7 +767,11 @@ func (a *App) UpdateChannelMemberNotifyProps(data map[string]string, channelId s
 
 func (a *App) DeleteChannel(channel *model.Channel, userId string) *model.AppError {
 	ihc := make(chan store.StoreResult, 1)
-	ohc := a.Srv.Store.Webhook().GetOutgoingByChannel(channel.Id, -1, -1)
+	outgoingHooks, err := a.Srv.Store.Webhook().GetOutgoingByChannel(channel.Id, -1, -1)
```

Comment:
we expect here to maintain the old behavior, async call (we need here explicit async, like in the next gorouting block plus the channel creation in the previous line. You can see an example of this in the example PR.